pyximport/pyximport.py

```diff
@@ -351,6 +351,9 @@ def __init__(self, pyxbuild_dir=None, inplace=False, language_level=None):
         self.uncompilable_modules = {}
         self.blocked_modules = ['Cython', 'pyxbuild', 'pyximport.pyxbuild',
                                 'distutils.extension', 'distutils.sysconfig']
+        # on OS X, distutils appears to shadow distutils.sysconfig
+        if os.name is 'posix' and os.uname()[0].lower().startswith('darwin'):
+            self.blocked_modules.append('distutils')
```

Comment:
I think it's reasonable to _always_ block all of distutils. Compiling the compiling infrastructure while it's being used just unnecessarily screams for trouble.

Reply:
good point; i wanted to be as surgical as possible, but this is cleaner and better